com.github.ransome1.sleek.yml

```diff
@@ -0,0 +1,60 @@
+app-id: com.github.ransome1.sleek
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '20.08'
+separate-locales: false
+command: sleek
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --device=dri
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+modules:
+  - name: sleek
+    build-options:
+      append-path: /usr/lib/sdk/node14/bin
+      env:
+        XDG_CACHE_HOME: /run/build/sleek/flatpak-node/cache
+        npm_config_nodedir: /usr/lib/sdk/node14
+    buildsystem: simple
+    build-commands:
+      - npm install --prefix=sleek --offline --cache=$FLATPAK_BUILDER_BUILDDIR/flatpak-node/npm-cache
+      - . flatpak-node/electron-builder-arch-args.sh; npm run --prefix=sleek --offline pack -- $ELECTRON_BUILDER_ARCH_ARGS
+      - mv sleek/dist/linux*unpacked /app/sleek
+      - install -Dm755 sleek.sh /app/bin/sleek
+    sources:
+      - type: git
+        url: https://github.com/ransome1/sleek.git
+        commit: 1bcdc188a9f47eb08116fe532bd8e2ba076b7b04
+        dest: sleek
+      - generated-sources.json
+
+      - type: script
+        commands:
+          - exec env TMPDIR=$XDG_CACHE_HOME zypak-wrapper /app/sleek/sleek
+        dest-filename: sleek.sh
+  - name: appdata
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 com.github.ransome1.sleek.appdata.xml /app/share/metainfo/com.github.ransome1.sleek.appdata.xml
+    sources:
+      - type: file
+        path: com.github.ransome1.sleek.appdata.xml
+  - name: desktop-entry
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 com.github.ransome1.sleek.desktop /app/share/applications/com.github.ransome1.sleek.desktop
+    sources:
+      - type: file
+        path: com.github.ransome1.sleek.desktop
+  - name: icon
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 com.github.ransome1.sleek.png /run/build/icon/com.github.ransome1.sleek.png
```

Comment:
you are installing in place which is why you have the build error.

The icon should be installed in `/app/share/icons/hicolor/128x128/apps/`.

Reply:
> you are installing in place which is why you have the build error.
> 
> The icon should be installed in `/app/share/icons/hicolor/128x128/apps/`.

I changed the path, but it didn't solve the problem, unfortunately.

Reply:
it did, it's a different build failure.